k8s/upgrades/README.md

```diff
@@ -4,12 +4,12 @@
 
 This document describes the steps for the following OpenEBS Upgrade paths:
 
-- Upgrade from 1.0.0 to any of 1.1.0, 1.2.0, 1.3.0, 1.4.0, 1.5.0, 1.6.0
-- Upgrade from 1.1.0 to any of 1.2.0, 1.3.0, 1.4.0, 1.5.0, 1.6.0
-- Upgrade from 1.2.0 to any of 1.3.0, 1.4.0, 1.5.0, 1.6.0
-- Upgrade from 1.3.0 to any of 1.4.0, 1.5.0, 1.6.0
-- Upgrade from 1.4.0 to 1.5.0, 1.6.0
-- Upgrade from 1.5.0 to 1.6.0
+- Upgrade from 1.0.0 to any of 1.1.0, 1.2.0, 1.3.0, 1.4.0, 1.5.0, 1.6.0, 1.7.0
+- Upgrade from 1.1.0 to any of 1.2.0, 1.3.0, 1.4.0, 1.5.0, 1.6.0, 1.7.0
+- Upgrade from 1.2.0 to any of 1.3.0, 1.4.0, 1.5.0, 1.6.0, 1.7.0
+- Upgrade from 1.3.0 to any of 1.4.0, 1.5.0, 1.6.0, 1.7.0
+- Upgrade from 1.4.0 to 1.5.0, 1.6.0, 1.7.0
+- Upgrade from 1.5.0 to 1.6.0, 1.7.0
```

Comment:
Add one more upgrade path `Upgrade from 1.6.0 to 1.7.0`

Reply:
Done. Thank you. I missed it.